.goreleaser.yaml

```diff
@@ -1,3 +1,4 @@
+version: 2
```

Comment:
It's needed to avoid this error
https://github.com/moneyforward/auriga/actions/runs/9889651283/job/27316188732